staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -195,4 +195,7 @@ type Interface interface {
 	GuaranteedUpdate(
 		ctx context.Context, key string, ptrToType runtime.Object, ignoreNotFound bool,
 		precondtions *Preconditions, tryUpdate UpdateFunc, suggestion ...runtime.Object) error
+
+	// Count returns number of keys under the pathPrefix.
+	Count(pathPrefix string) (int64, error)
```

Comment:
name the parameter `key` to make it clear it matches what is passed into List, etc?

Reply:
and include `ctx context.Context` as the first arg and plumb into the etcd call, like the other storage calls

Reply:
Done, though I think it may be a bit misleading. The list one is misleading as well though:)